start.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+
+
+# Install and upgrade
+echo "Installing build-essential..."
+apt-get update
+apt-get install -y build-essential
```

Comment:
I think we should probably remove all "-y" and explicitly tell the user "You don't have xyz installed, do you want to install it now?"...claude suggests something like this to check if someone has build-essentials installed: 

```bash
if ! dpkg -s build-essential &> /dev/null; then
    echo "build-essential is not installed. Please install it using: sudo apt-get install build-essential"
    exit 1
fi
```